brownie/network/rpc.py

```diff
@@ -81,8 +87,15 @@ def launch(self, cmd: str, **kwargs: Dict) -> None:
         kwargs.setdefault("evm_version", EVM_DEFAULT)  # type: ignore
         if kwargs["evm_version"] in EVM_EQUIVALENTS:
             kwargs["evm_version"] = EVM_EQUIVALENTS[kwargs["evm_version"]]  # type: ignore
+        kwargs = _validate_cmd_settings(kwargs)
         for key, value in [(k, v) for k, v in kwargs.items() if v]:
-            cmd += f" {CLI_FLAGS[key]} {value}"
+            try:
+                cmd += f" {CLI_FLAGS[key]} {value}"
+            except KeyError:
+                warnings.warn(
```

Comment:
In the spirit of #499, this should probably be a brownie-specific warning rather than a builtin.